src/features/promise-combinators.md

```diff
@@ -90,8 +90,8 @@ We kick off a computationally expensive task that might take a long time, but we
 ## `Promise.allSettled`
 
 <feature-support chrome="76"
-                 firefox="no https://bugzilla.mozilla.org/show_bug.cgi?id=1549176"
-                 safari="no https://bugs.webkit.org/show_bug.cgi?id=197600"
+                 firefox="68"
```

Comment:
This change is incorrect. https://bugzilla.mozilla.org/show_bug.cgi?id=1549176 is still not fixed (i.e. it’s still Nightly-only). Please revert this part

Reply:
done